src/sage/topology/moment_angle_complex.py

```diff
@@ -0,0 +1,781 @@
+"""
+Moment-angle complexes
+
+AUTHORS:
+
+- Ognjen Petrov (2023-06-25)
+"""
+
+# ****************************************************************************
+#       Copyright (C) 2013 Ognjen Petrov <ognjenpetrov@yahoo.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# ****************************************************************************
+
+from sage.misc.cachefunc import cached_method
+from sage.misc.lazy_attribute import lazy_attribute
+from sage.homology.homology_group import HomologyGroup
+from sage.rings.integer_ring import ZZ
+from sage.rings.rational_field import QQ
+from sage.structure.sage_object import SageObject
+from sage.structure.unique_representation import UniqueRepresentation
+from .cubical_complex import CubicalComplex, cubical_complexes
+from .simplicial_complex import SimplicialComplex, copy
+from sage.topology import simplicial_complex_catalog as simplicial_complexes
+from itertools import combinations
+
+# Future TODO's:
+# - add latex_name parameter or something similar?
+# - different way of computing euler characteristic?
+# - bigraded betti numbers? (page 161 buchstaber panov)
+# - explicitly state the vertices for construction
+# - polyhedral products and real moment-angle complexes
+# - golod decomposition
+# - return for odd dimensional simplicial complexes in golod_decomposition?
+
+
+# maybe make hidden?
```

Comment:
Let's remove these comments. Anything you would want to make a formal todo, I would add to the module-level doc as a `.. TODO::`.

Reply:
Thank you, I agree. How should all of these items be listed within the ``TODO`` block, using `-` for each item, or just have separate paragraphs for each one?

Reply:
Most if not all of them should simply be issues on GH (if even this). IMO, none of them are things to include in the doc, but I am applying a very high standard for this as none of them are needed for a "complete" (in some sense) implementation.

Reply:
So I should remove all of the TODO's and just have GH issues?

Reply:
I think that might be best since that will be targeted only to developers (which perhaps you will implement soon anyways).